recipes/benchmark/all/conanfile.py

```diff
@@ -52,6 +49,10 @@ def validate(self):
         if Version(self.version) < "1.7.0" and is_msvc(self) and self.info.options.shared:
             raise ConanInvalidConfiguration(f"{self.ref} doesn't support msvc shared builds")
 
+    # TODO: it is required for several CI environment(ex. linux gcc 11)
+    def build_requirements(self):
+        self.tool_requires("cmake/3.16.3")
```

Comment:
not like this, conditional based on version please, and don't hesitate to use a recent CMake version.